functions/src/days-before-campaign-ends.test.ts

```diff
@@ -11,6 +11,16 @@ test("6 days remain", t => {
   t.is(actual, 6);
 });
 
+test("6 days remain 2", t => {
```

Comment:
The title of the test does not reflect it's behavior. What is important for this test is that "now" is not in the same month as the end of the campaign. I suggest "21 days remain, end of campaign is next month".